lzero/model/gobigger/network/scatter_connection.py

```diff
@@ -0,0 +1,89 @@
+from typing import Tuple
```

Comment:
add this part of codes into DI-engine

Reply:
https://github.com/opendilab/DI-engine/pull/680